/**:
  ros__parameters:
    cmd_freq: 100.0  # Hz of platform commands send
    info_freq: 10.0  # Hz of platform info publish
    enable_advanced_sensing: False  # Enable the possibility to access to all cameras and sensor
    publish_camera: True # Advance sensing must be enabled
    enable_mop_channel: False